fly/commands/completion.go

```diff
@@ -0,0 +1,27 @@
+package commands
+
+import "fmt"
+
+type CompletionCommand struct {
+	Bash BashCompletionCommand `command:"bash"`
```

Comment:
Is `foo completion bash` a common convention in other CLIs? I only ask because the pattern elsewhere in `fly` is to use flags instead of arguments. So it'd be something like `fly completion --shell bash` instead, with `--shell` being something like this:

```suggestion
	Shell string `long:"shell" required:"true" choice:"bash"` // add more choices later
```

Reply:
`kubectl` and `helm` have `completion bash`. `pip` has `completion --bash`, while `pipenv` has just `--completion` with autodetecting the running shell, and `terraform` does `-install-autocomplete`. I guess the convention is not as common as I thought :)

Reply:
Thanks for checking. :slightly_smiling_face: